routes/events.js

```diff
@@ -39,18 +39,18 @@ module.exports = function(app) {
         var auth_token = req.get('Authorization');
         jwt.verify(auth_token, app.get('superSecret'), function(err, userId) {
             if (err) {
-                console.log(err);
-                res.sendStatus(500);
+                console.log('Error verifying auth token:', err);
+                res.status(500).send(err);
             }
             User.findById(userId, function(error, user) {
                 if (error) {
-                    console.log(error);
-                    res.sendStatus(500);
+                    console.log('Error finding user by ID:', error);
+                    res.status(500).send(error);
                 }
                 eventServices.findAllEventsInTier(user.tier, function(err, events) {
                     if (err) {
```

Comment:
'err' is being defined twice in this context (first from the authentication callback), probably want to use a different variable name here